cloud/google/cmd/gce-machine-controller/app/controller.go

```diff
@@ -60,10 +61,24 @@ func StartMachineController(server *options.MachineControllerServer, shutdown <-
 	if err != nil {
 		glog.Fatalf("Could not create config watch: %v", err)
 	}
+
+	sshUser, err := ioutil.ReadFile(server.SSHUserPath)
+	if err != nil {
+		glog.Fatalf("Could not read user file: %v", err)
+	}
+
+	sshPublicKey, err := ioutil.ReadFile(server.SSHUserPath)
```

Comment:
Sorry -- once you gave the justification for why private key needed to be a file it may make more sense to keep the public key as a file as well

The user is a bit odd to me -- it seems like shouldn't be considered a secret?